ethereum-types/Cargo.toml

```diff
@@ -15,6 +15,7 @@ primitive-types = { path = "../primitive-types", version = "0.10", features = ["
 impl-serde = { path = "../primitive-types/impls/serde", version = "0.3.0", default-features = false, optional = true }
 impl-rlp = { path = "../primitive-types/impls/rlp", version = "0.3", default-features = false, optional = true }
 impl-codec = { version = "0.5.0", path = "../primitive-types/impls/codec", default-features = false, optional = true }
+scale-info-crate = { package = "scale-info", version = ">=0.9, <2", features = ["derive"], default-features = false, optional = true }
```

Comment:
```suggestion
```

Reply:
I left it there for now because I expect I'll have to derive it for some more types in this crate. If that turns out not to be the case, I'll remove this.